tests/app/template/test_rest.py

```diff
@@ -100,22 +99,25 @@ def test_should_create_a_new_template_for_a_service(
     assert sorted(json_resp['data']) == sorted(template_schema.dump(template))
 
 
-@pytest.mark.xfail(reason='Mislabelled for route removal, fails when unskipped', run=False)
+@pytest.mark.skip(reason='TODO #2336 - Fail due to orphaned User object')
+@pytest.mark.serial
```

Comment:
ditto not serial

Reply:
Updated and remove skip. 

Confirm test passes locally and on Github Jobs.